state-chain/runtime/src/chainflip/solana_elections.rs

```diff
@@ -990,3 +990,181 @@ impl BenchmarkValue for SolanaVaultSwapsSettings {
 		}
 	}
 }
+
+#[cfg(test)]
+mod tests {
+	use electoral_systems::solana_swap_accounts_tracking::SolanaVaultSwapsVote;
+	use pallet_cf_elections::electoral_system::{mock::DummyAccess, ConsensusVote, ConsensusVotes};
+
+	use super::*;
+
+	pub const NEW_ACCOUNT_1: SolAddress = SolAddress([1u8; 32]);
+	pub const NEW_ACCOUNT_2: SolAddress = SolAddress([2u8; 32]);
+	pub const NEW_ACCOUNT_3: SolAddress = SolAddress([3u8; 32]);
+
+	pub const CLOSED_ACCOUNT_1: SolAddress = SolAddress([4u8; 32]);
+	pub const CLOSED_ACCOUNT_2: SolAddress = SolAddress([5u8; 32]);
+
+	pub const DUMMY_VALIDATOR: [u8; 32] = [0u8; 32];
+
+	#[test]
+	fn test_solana_vault_swaps_consensus() {
```

Comment:
would have a look at some tests for the other electoral systems for some ideas on how to tidy this up a bit.
e.g. state-chain/pallets/cf-elections/src/electoral_systems/tests/liveness.rs
state-chain/pallets/cf-elections/src/electoral_systems/tests/monotonic_change.rs